.github/dependabot.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  reviewers:
+  - jglick
+  - oleg-nenashev
+  - batmat
+  ignore:
+  - dependency-name: jcl-over-slf4j
+    versions:
+    - ">= 0"
```

Comment:
Is this actually necessary in the new syntax, or just a dumb translation?

Reply:
dumb translation. I will clean it up during the next update